mmv1/templates/terraform/encoders/compute_backend_bucket.go.erb

```diff
@@ -0,0 +1,46 @@
+// This custom encoder helps prevent sending 0 for clientTtl, defaultTtl and
+// maxTtl in PATCH calls when unset in the provider (doing so results in an API
+// level error)
+c, cdnPolicyOk := d.GetOk("cdn_policy")
+
+// Only apply during update
+if !cdnPolicyOk {
+	return obj, nil
+}
+
+// Don't do anything if we're not PATCHing
+if obj["cdnPolicy"] == nil {
+	return obj, nil
+}
+
+cdnPolicies := c.([]interface{})
```

Comment:
Nit: it might make sense to rephrase these as currentCDNPolicies and futureCDNPolicies. It's difficult to understand the logic implicitly otherwise.

Reply:
done